src/python/python/xml.py

```diff
@@ -0,0 +1,617 @@
+import numpy as np
+import os
+from shutil import copy2
+import math
+
+class Files:
+    MAIN = 0
+    MATS = 1
+    GEOM = 2
+    EMIT = 3
+    CAMS = 4
+    #TODO: Volumes
+
+class WriteXML:
+    '''
+    File Writing API
+    '''
+
+    defaults = { #arg name: default name
+        'sample_count': 'spp',
+        'width': 'resx',
+        'height': 'resy'
+    }
+
+    def __init__(self, path, split_files=False):
+        from mitsuba.core import PluginManager
```

Comment:
Make sure we have set a variant? Without overwriting the current variant (if any) of course